frontend_build/src/read_bundle_plugins.js

```diff
@@ -105,7 +105,7 @@ var readBundlePlugin = function(base_dir) {
       // If this is not the core bundle, then we need to add the external library mappings.
       bundle.externals = _.extend({}, externals, lib_externals);
     } else {
-      bundle.externals = externals;
+      bundle.externals =  _.extend({}, externals, { kolibri: core_bundle.output.library });
```

Comment:
This is the change that is causing the test failures, and this was an inadequate solution, so this should be reverted, @ralphiee22.